website/content/docs/concepts/policies.mdx

```diff
@@ -800,32 +800,36 @@ authenticated user.
+| [sys/auth/:path/tune](/vault/api-docs/system/auth#tune-auth-method)                                                                                    | GET, POST, DELETE | Manage the auth methods (enable, read, delete, and tune)                                                                                            |
 | [sys/config/auditing/request-headers](/vault/api-docs/system/config-auditing)                                                                          | GET               | List the request headers that are configured to be audited                                                                                          |
-| [sys/config/auditing/request-headers:name](/vault/api-docs/system/config-auditing)                                                                     | GET, PUT, DELETE  | Manage the auditing headers (create, update, read and delete)                                                                                       |
-| [sys/config/cors](/vault/api-docs/system/config-cors)                                                                                                  | GET, PUT, DELETE  | Configure CORS setting                                                                                                                              |
-| [sys/config-ui](/vault/api-docs/system/config-ui)                                                                                                      | GET               | Configure the UI settings                                                                                                                           |
-| [sys/internal/specs/openapi](/vault/api-docs/system/internal-specs-openapi)                                                                           | GET               | Generate an OpenAPI document of the mounted backends                                                                                                |
+| [sys/config/auditing/request-headers/:name](/vault/api-docs/system/config-auditing)                                                                    | GET, POST, DELETE | Manage the auditing headers (create, update, read and delete)                                                                                       |
+| [sys/config/cors](/vault/api-docs/system/config-cors)                                                                                                  | GET, POST, DELETE | Configure CORS setting                                                                                                                              |
+| [sys/config-ui/headers](/vault/api-docs/system/config-ui)                                                                                              | GET               | Configure the UI settings                                                                                                                           |
```

Comment:
Path: Should be `sys/config/ui/headers` - slash not dash

Verb: Actually LIST

Description: e.g. `List custom HTTP headers configured to be served with the UI`

Reply:
Added list and fixed slash/dash to upcoming commit